sql/create-db.sql

```diff
@@ -93,3 +93,16 @@ CREATE TABLE IF NOT EXISTS amiusage (
   lastused TIMESTAMPTZ NOT NULL, -- most recent usage
   PRIMARY KEY(id, region)
 );
+
+-- Elastic Block Store (EBS) Volumes uasge
+-- We are only interested in storing the total number of volumes and the total number
+-- of GBs per type/region from the most recent housekeeping sweep.
+CREATE TABLE IF NOT EXISTS ebsusage (
+  region VARCHAR(128) NOT NULL, -- ec2 region
+  volumetype VARCHAR(128) NOT NULL,  -- volume type (e.g gp2, st1)
+  state VARCHAR(128) NOT NULL CHECK (state in ('active', 'unused')), -- state of aggregated volumes
```

Comment:
if these are active/unused, they don't need 128 bytes :)

Reply:
Good point! I will make those much smaller :)

Reply:
@jhford seemed to imply that's unnecessary?

https://github.com/taskcluster/ec2-manager/pull/29#discussion_r142249195